scripts/info.js

```diff
@@ -0,0 +1,24 @@
+var information = [];
+
+function Info (catagories) {
+  this.infoTitle = catagories.title; //Title of info
+  this.summery = catagories.summery; //Summery of info
+}
+
+Info.prototype.toHtml = function() {
+  var $infoChunk = $('section').clone();
+  $infoChunk.find('title').html(this.infoTitle);
+  $infoChunk.find('.blogSummery').html(this.summery);
+  $infoChunk.removeAttr('class');
+  console.log('Prototyping: '+this.infoTitle);
+  return $infoChunk;
+};
+
+soccerBlog.forEach(function(element){
+  information.push(new Info(element));
+});
+
+information.forEach(function(soccerPost){ // Get it?   Soccer... Post...
+  $('section').append(soccerPost.toHtml());
```

Comment:
I think you want to append to 'main' instead of 'section'. The soccerPost HTML is itself a 'section' tag, so by appending to all 'section' tags, you are creating a loop of appends.